build/gen_stub.php

```diff
@@ -576,11 +576,23 @@ public function getValue(): string {
 
     public function getVariableName(): string {
         $value = $this->getValue();
-        if ($value === null || strlen($value) === 0 || $value[0] !== '$') {
-            throw new Exception("@$this->name not followed by variable name");
+        if ($value === null || strlen($value) === 0) {
+            throw new Exception("@$this->name doesn't have any value");
         }
 
-        return substr($value, 1);
+        $matches = [];
+
+        if ($this->name === "param") {
+            preg_match('/^\s*[\w\|\\\]+\s*\$(\w+).*$/', $value, $matches);
```

Comment:
```suggestion
            preg_match('/^\s*[\w\|\\\\]+\s*\$(\w+).*$/', $value, $matches);
```
Three backslashes works, but is pretty confusing :) Makes it look like the `]` is getting escaped.

Reply:
Alright, I'll fix it after applying the other commit to master as soon as CI is green.